drivers/input/touchscreen/s6sy761.c

```diff
@@ -326,7 +326,7 @@ static int s6sy761_power_on(struct s6sy761_data *sdata)
 
 	/* for some reasons the device might be stuck in the bootloader */
 	if (ret != S6SY761_BS_APPLICATION)
-		return -ENODEV;
+		dev_warn(&sdata->client->dev, "Unknown status: %d\n", ret);
```

Comment:
I would suggest splitting touchscreen in dts, as it requires an extra patch to work.
Then we can upstream the touchscreen once it gets fixed.

Reply:
@wonderfulShrineMaidenOfParadise I think this could be because of missing regulator on gpio88 and reset at gpio96 and driver is fine. Didn't notice any problem with it before though.

Reply:
gpio 88 is BTP_LDO_EN and gpio is BTP_RST_N. I have no idea what BTP is myself. Is it something relevant?